### PR TITLE
Frontend react

### PR DESCRIPTION
**Pull request by PrangonGhose**
### Changes made in this PR
- Created `Redux` store and reducers to fetch random generated message from the `Rails API` used in [hello-rails-back-end](https://github.com/PrangonGhose/hello-rails-back-end) repository.
- Used the Redux store and React routes to load the returned data from the API to the home page of the app.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,22 @@
...
-        </a>
-      </header>
-    </div>
+    <>
+      <h1>
+        Random Greeting:
+        {newVar}
+      </h1>
+    </>
   );
```
Comment (jerryowusu) [inline src/App.js]: - [ ] As part of the project requirements, you are suppose to create the Greeting component that will display a greeting. Set it up as a route in your App component. 

- [ ] You should also display the random greeting in your Greeting component. 

However, I see that you have done all this in the app.js component. 

- [ ] You should also create the App component with react-router such that it has a route that leads to the Greeting component

Kindly update your code to reflect the project requirements as shown below :point_down: 


Screenshot from 2023-05-11 17-52-52

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @PrangonGhose 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

noooo


## Highlights
:heavy_check_mark: Used gitflow
:heavy_check_mark: Redux store set up
:heavy_check_mark: Redux action and reducer added

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by PrangonGhose** [comment]
### Changes made after first review
- Created `Greetings` component as per the requirement
- Created `App.js` with `react-router`

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @PrangonGhose👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Good job with requirements. ✔️
- Gitflow is followed. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**